### PR TITLE
Add test_cli

### PR DESCRIPTION
**Pull request by m-rey**

### CODE REVIEW

File: tests/test_cli.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import pytest
+import leo
+
+
+def test_cli():
+    args = leo.parse(["-l", "de", "apfel"])
+    print(args)
```
Comment (tomschr) [inline tests/test_cli.py]: Is that a art or can we remove that? :wink:

**Review by tomschr — COMMENTED** [review]
Just one minor comment, otherwise ok. :+1:

File: tests/test_cli.py (hunk 1 above)
Comment (m-rey) [reply tests/test_cli.py]: oops, my bad

**Comment by m-rey** [comment]
change already applied in #7.

**Pull request closed without merging**